lib/init/autoconfig.js

```diff
@@ -333,7 +333,7 @@ function extendFromRecommended(config) {
             delete newConfig.rules[ruleId];
         }
     });
-    newConfig.extends = RECOMMENDED_CONFIG_NAME;
+    newConfig.extends = [RECOMMENDED_CONFIG_NAME];
```

Comment:
I think this will overwrite any other plugins that are already set [here](https://github.com/eslint/eslint/blob/master/lib/init/config-initializer.js#L289-L297). Can we instead push to `newConfig.extends`? It will get normalized in the end [here](https://github.com/eslint/eslint/blob/master/lib/init/config-initializer.js#L333-L338) anyway.

Reply:
@kaicataldo 
Yes. I'll add test cases. thanks. :)